pkg/appStore/deployment/service/AppStoreDeploymentService.go

```diff
@@ -978,6 +1031,25 @@ func (impl AppStoreDeploymentServiceImpl) UpdateInstalledApp(ctx context.Context
 		impl.logger.Errorw("fetching error", "err", err)
 		return nil, err
 	}
+	installedAppVersionHistory := &repository.InstalledAppVersionHistory{
+		InstalledAppVersionId: installAppVersionRequest.Id,
+		ValuesYamlRaw:         installAppVersionRequest.ValuesOverrideYaml,
+		StartedOn:             time.Now(),
+		Status:                pipelineConfig.WorkflowInProgress,
+		AuditLog: sql.AuditLog{
+			CreatedOn: time.Now(),
+			CreatedBy: installAppVersionRequest.UserId,
+			UpdatedOn: time.Now(),
+			UpdatedBy: installAppVersionRequest.UserId,
+		},
```

Comment:
after seeing this at multiple places, I feel it should be extracted out as a method  taking in arguments for variable fileds